### PR TITLE
feature1 implementation

### PR DESCRIPTION
**Pull request by oryades**
Team, please review changes.

### CODE REVIEW

File: script.sh [hunk 1]
```diff
@@ -4,3 +4,7 @@ Version for my_branch
 Commit 1
 Commit 3 on main branch
 Commit 2 on my_branch
+
+function feature1 {
```
Comment (Tom-chack) [inline script.sh:8]: Shorter way is:
```
feature1(){
}
```

**Review by satadevosyan — APPROVED** [review]

**Review by Tom-chack — COMMENTED** [review]
All is ok

**Review by Tom-chack — APPROVED** [review]

**Review by durmm — APPROVED** [review]

File: script.sh (hunk 1 above)
Comment (hakob-mnatsakanyan-sada) [reply script.sh:8]: Good

**Comment by a3925ha** [comment]
Empty function ?
Hmmm....

**Review by Morgana90 — APPROVED** [review]

**Review by Artghochikyan — APPROVED** [review]

**Review by a3925ha — APPROVED** [review]

**Pull request merged**